### ISSUE TITLE
formatting on console.log of look logging

### ISSUE THREAD

**Issue by longhorn09** [longhorn09/lorebot#8]
need space after moment().format() timestamp before colon and padEnd(30)

```
2017-06-27 23:53:21 : patrick                        !stat mantooth
2017-06-28 00:19:25: eddiejerky                     Logged 'Tterror'
2017-06-28 00:30:29 : normstorm                      !who Tterror
```

**Comment by longhorn09** [comment]
Believe already fixed in ParseEqLook, need to validate before closing
```    
CreateUpdatePerson(charName,light,ring1,ring2,neck1,neck2,body,head,legs,feet,arms,slung,
                      hands,shield,about,waist,pouch,rwrist,lwrist,primary,secondary,held,both,pSubmitter,null, (arg) => {
                      console.log(`${moment().format(MYSQL_DATETIME_FORMAT)} : ${pSubmitter.padEnd(30)} Logged '${arg}'`);
                    });

```

**Comment by longhorn09** [comment]
addressed in 533c13f
